en/controllers/components/csrf.rst

```diff
@@ -15,6 +15,12 @@ request data & cookie value. If either is missing or the two values mismatch the
 component will throw a
 :php:class:`Cake\\Network\\Exception\\InvalidCsrfTokenException`.
 
+.. note::
+    You should always verify the HTTP method being used before doing with
+    side-effects. You should :ref:`check the HTTP method <check-the-request>` or
```

Comment:
You're right to be confused, because this makes no sense. I should read what I write before pushing it.